app/templates/_CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# How to contribute
+
+To maintain high traceability and quality, all contributions after the initial setup need to adhere to the following guidelines:
+
+- [Create a github issue](https://github.com/liferay-lexicon/<%= repoName %>/issues/new) explaining the issue to be fixed or use feature to be implemented.
```

Comment:
This still contains the organization `liferay-lexicon`... that should be configurable as well, right?